RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 * Redirect to username/password login when WordPress.com reports email login not allowed
 * Updated notifications list to Material guidelines
 * Updated snackbar design to Material guidelines
+* Added ability to toggle to compact view type for blog posts list
```

Comment:
Note for myself - we need to move this line to 12.4 when we merge develop into this branch.